packages/cardano-graphql/src/Schema/types/Transaction/AuxiliaryData/Metadatum.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-use-before-define */
+import { Directive, Field, Int, ObjectType, createUnionType } from 'type-graphql';
+import { NotImplementedError } from '@cardano-sdk/core';
+
+// Review: I think integers will be int32 or int64. We left it as bigint in core types,
```

Comment:
Maybe best to just log this as a follow-up task for QA to confirm this for us. Is this currently blocking a merge?

Reply:
Don't think it's blocking, there's no server yet anyway. I'll log the task